backend/src/controllers/person.controller.ts

```diff
@@ -16,40 +16,66 @@ export const createPerson: POST = async (
   next: NextFunction,
 ): Promise<void> => {
   logger.info('POST /persons/create request from frontend');
```

Comment:
Could we update this log to just `POST /persons` rather than including the create to reflect the API design? Other than that the change looks great!

Reply:
No worries, should be fixed now 👍